auth/api-client/restrict_api_key_android.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START auth_cloud_restrict_api_key_android]
+
+from google.cloud import api_keys_v2
+from google.cloud.api_keys_v2 import Key
+
+
+def restrict_api_key_android(project_id: str, key_id: str) -> Key:
+    """
+    Restricts an API key based on android applications.
+
+    Specifies the Android application that can use the key.
+
+    TODO(Developer): Replace the variables before running this sample.
+
+    Args:
+        project_id: Google Cloud project id.
+        key_id: ID of the key to restrict. This ID is auto-created during key creation.
+        This is different from the key string. To obtain the key_id,
+        you can also use the lookup api: client.lookup_key()
+
+    Returns:
+        response: Returns the updated API Key.
```

Comment:
This style is fine to keep, but usually unless there's more than one, they don't necessarily need to be described with their types.

Instead it could be
```
Returns:
    The updated API `Key` response.
```

Reply:
Ack. Thanks Dan.